lib/parsers/gopkg-parser.ts

```diff
@@ -0,0 +1,64 @@
+
```

Comment:
This file is not the same format as the 'gomod-graph-parser'
let's add a prettier to this service (not in this pr)

Reply:
We can't add prettier atm cause it's node 10+ and we need to support node 8 +.\
Tested this earlier today and hit a wall.

See [prettier engine spec](https://github.com/prettier/prettier/blob/master/package.json#L14)

Reply:
prettier is a dev dependency and is only used in build
this should be ok afaik

Reply:
I understand, but the env *running* this is node 8, which fails to run the build phase

Reply:
we can use a lower version of prettier that supports node 8

Reply:
we can use a lower version of prettier that supports node 8 e.g. `v1.18.2`